src/Core/Silk.NET.Core/Loader/DefaultPathResolver.cs

```diff
@@ -276,14 +276,18 @@ private string GetUserDirectory()
         private List<string> GetAllRuntimeIds(string currentRid, DependencyContext ctx)
         {
             var allRiDs = new List<string>();
-            allRiDs.Add(currentRid);
-            if (!AddFallbacks(allRiDs, currentRid, ctx.RuntimeGraph))
+
+            if (ctx is not null) // prevent null reference exception on net6.0-android where DependencyContext.Default is null
```

Comment:
This line exceeds 120 characters.
```suggestion
            // prevent null reference exception on net6.0-android where DependencyContext.Default is null
            if (ctx is not null)
```

(you can press the "Apply Suggestion" button and it'll commit it for you)